zimui/src/style.css

```diff
@@ -10,3 +10,10 @@
   margin-right: auto;
   margin-left: auto;
 }
+.tooltip {
```

Comment:
Why do we need to put these values in the global `style.css`?

Reply:
1. To have a consistent bootstrap tooltip (default background color is black which doesn't look good with our black buttons) throughout the app.